crates/nu-command/Cargo.toml

```diff
@@ -12,7 +12,7 @@ doctest = false
 
 [dependencies]
 nu-data = { version = "0.27.1", path = "../nu-data" }
-nu-engine = { version = "0.27.1", path = "../nu-engine" }
+nu-engine = { version = "0.27.1", path = "../nu-engine", features = ["which"]}
```

Comment:
I don't think you need the `features = ["which"]` here. I think everything can drive from the main Cargo.toml file.